profiles/models.py

```diff
@@ -0,0 +1,28 @@
+"""
+Models for user profile
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+class Profile(models.Model):
+    """
+    A user profile
+    """
+    user = models.ForeignKey(User)
+    edx_username = models.TextField()
```

Comment:
It seems like we can't rely on the two usernames always being identical, but you probably know better than me. When the user logs in via OAuth is the User object created automatically?